### ISSUE TITLE
All training parameters of BasNet training from DeepUSPS labels (unsupervised saliency setting) are not provided

### ISSUE THREAD

**Issue by zadaianchuk** [wvangansbeke/Unsupervised-Semantic-Segmentation#27]
Hi @wvangansbeke
Thanks a lot for sharing the weights of pretrained BasNet! 
Unfortunately, I anyway need to train BASNet from scratch on DeepUSPS outputs on MSRA10k/MSRA-B. 
Can you please **provide all the hyperparameters that were changed from default values** during BasNet training on MSRA? Also, exact parameters that were used for DeepUSPS would be useful. E.g. can you specify what network was used as a DeepUSPS backbone? Was it pretrained with any other dataset (e.g. CityScapes or ImageNet)? This would make the whole pipeline of unsupervised saliency bootstrapping fully reproducible.

**Comment by wvangansbeke** [comment]
In short, we refer to the original repositories for more details. All training parameters can be kept the same to obtain our results.

When you make changes (e.g. initialization, architecture) only small changes to the training parameters are required. For example, the publically available DeepUSPS model additionally relies on the Cityscapes dataset but this only slightly influences the final results (see DeepUSPS paper). To keep this repo structured and easy to follow you can use our unsupervised model or follow the instructions provided in the original repos with your preferred changes.
